src/test/common/test_type_check_decorator.py

```diff
@@ -35,7 +35,7 @@ def bool_arg(b: bool):
 
         with pytest.raises(UnsupportedTypeError) as e:
             bool_arg(10)
-        assert "'b' must be 'bool'. The given argument type is 'int'" in str(e.value)
+        assert "'b' must be 'bool_parsing'. The given argument type is 'int'" in str(e.value)
```

Comment:
I would prefer to keep this warning statement as it was before.
The pydantic ValidationError type is listed, but the information the user wants is the correct variable type.
The information about the type the user wants is `bool`, not `bool_parsing``.

Reply:
I have confirmed that this has been corrected in #312.